### PR TITLE
library: fix a bug in ceph_config module

### PR DESCRIPTION
**Pull request by guits**
`ceph config get` doesn't allow getting value when entity is 'global' so it makes the module fail. Indeed, the module first tries to get the current value of the option for idempotency concern. The idea is to check in `ceph config dump` instead.

Closes: https://bugzilla.redhat.com/show_bug.cgi?id=2188319

### CODE REVIEW

**Comment by guits** [comment]
jenkins test el8-functional

**Comment by guits** [comment]
jenkins test el8-functional

File: library/ceph_config.py [hunk 1]
```diff
@@ -135,15 +149,15 @@ def main() -> None:
     startd = datetime.datetime.now()
     changed = False
 
-    rc, cmd, out, err = get_or_set_option(module, 'get', who, option, value)
-    if rc:
-        fatal(message=f"Can't get current value. who={who} option={option}", module=module)
+    rc, cmd, out, err = get_config_dump(module)
```
Comment (adk3798) [inline library/ceph_config.py:152]: before this change when the action was `get`, `rc, cmd, out, err` would have ended up set as the result of the `ceph config get` command. With this change, it's now from the result of the `ceph config dump`. Are we doing anything with that out that means that would be an issue? I.e. if we were returning that output to show the user the value of the config option they'd now get a big json blob of all of the options.

Comment (guits) [reply library/ceph_config.py:152]: it shouldn't be an issue, `out` is set L158 in the case where the value is already set or L160 if a new value had to be set.

Comment (adk3798) [reply library/ceph_config.py:152]: yeah, but line 158 only runs `if action == 'set'`. My concern was for when the action is `get`.

Comment (guits) [reply library/ceph_config.py:152]: true

**Review by adk3798 — APPROVED** [review]

**Pull request merged**